samples/subsys/tracing/sample.yaml

```diff
@@ -16,6 +16,8 @@ tests:
       - CONFIG_TRACING=y
       - CONFIG_TRACING_CPU_STATS=y
   tracing.osawareness.openocd:
+    integration_platforms:
+      - frdm_k64f
```

Comment:
Recommend adding boards to represent other architectures. OS awareness has some differences across arches

Reply:
So from an `integration_platforms` platform perspective this is only a build test since testing openocd w/qemu isn't something we can do.  So is there any particular value in building this on the qemu platforms?

Reply:
There are build-time conditionals for the various arches, and we've at at least one bug that I can remember around them.
https://github.com/zephyrproject-rtos/zephyr/blob/2dce6bff7f0c8106ceae16c783c387d2f608f7d2/subsys/debug/openocd.c#L51-L83